atlp-rwanda/atlp-pulse-mobile
# Fix Preference Page Icons

### What does this PR do?
This PR updates the trainee preference settings page to maintain consistency by replacing all buttons or clickables with Chevron icons.
### Description of Task to be completed?

- Replace the forward arrow icons on the settings page with Chevron icons for better UI consistency.
- Ensure that the new Chevron icons are interactive and reflect the current colorScheme (light or dark mode).
- Update the push notification section to include expand/collapse functionality, toggling between a downward and forward Chevron icon.

### How should this be manually tested?

1. Navigate to the trainee preference settings page.

2. Verify that all forward arrows have been replaced by Chevron icons.

3. Test the push notification section:

- Click the Chevron icon to expand the section and reveal the switch.
- Click the Chevron icon again to collapse the section.

4. Ensure the Chevron icons adjust their color dynamically based on the active colorScheme.

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker/Trello stories?
N/A
### Screenshots


![image](https://github.com/user-attachments/assets/b1f31e64-f29b-49ef-abd1-e8f287d0f5dd)

![image](https://github.com/user-attachments/assets/4a27533a-396c-48aa-b80a-f290a9a29188)
